.github/workflows/test.yml

```diff
@@ -14,14 +14,33 @@ jobs:
     steps:
       - name: git checkout
         uses: actions/checkout@v3
+
       - name: Install Nix
         uses: DeterminateSystems/nix-installer-action@main
-      - uses: DeterminateSystems/magic-nix-cache-action@main
-      - uses: nicknovitski/nix-develop@v1
+
+      - name: Setup Nix cache
+        uses: DeterminateSystems/magic-nix-cache-action@main
+
+      - name: Setup Nix development environment
+        uses: nicknovitski/nix-develop@v1
         with:
           arguments: bsf/.#devShell
-      - name: run tests
+
+      - name: Set up Go
```

Comment:
We don't need this actually. We have all these dependencies defined in bsf.hcl, so its possible to access these binaries directly without adding new actions.

Reply:
removing the entire Go Setup?